runtime/php-intermediary.Dockerfile

```diff
@@ -411,11 +411,11 @@ RUN set -xe \
         --with-pdo-mysql=shared,mysqlnd \
         --with-mysqli \
         --enable-pcntl \
-        --enable-zip \
+        --with-zip \
         --enable-bcmath \
         --with-pdo-pgsql=shared,${INSTALL_DIR} \
         --enable-intl=shared \
-        --enable-opcache-file \
+        # --enable-opcache-file \ replaced with opcache.file_cache INI directive
```

Comment:
You can just drop this, the option only makes sense in the `--disable` form, as it was enabled by default.